docs/docs/path_algorithm.md

```diff
@@ -0,0 +1,406 @@
+
+`algo.SPpaths` and `algo.SSpaths` can solve a wide range of real-world problems, where minimal-weight paths need to be found. algo.SPpaths finds paths between a given pair of nodes, while algo.SSpaths finds paths from a given source node. Weight can represent time, distance, price, or any other measurement. A bound can be set on another property (e.g., finding a minimal-time bounded-price way to reach from point A to point B). Both algorithms are performant and have low memory requirements.
+
+For both algorithms, you can set:
+
+* A list of relationship types to traverse (`relTypes`).
+
+* The relationships' property whose sum you want to minimize (`weight`).
+
+* A optional relationships' property whose sum you want to bound (`cost`) and the optional bound (`maxCost`).
+
+* An optional bound on the path length - the number of relationships along the path (`maxLen`).
+
+* The number of paths you want to retrieve: either all minimal-weight paths (`pathCount` is 0), a single minimal-weight path (`pathCount` is 1), or _n_ minimal-weight paths with potentially different weights (`pathCount` is _n_).
+
+This topic explains which problems you can solve using these algorithms and demonstrates how to use them.
+
+Let's start with the following graph.
+
+![Road network](../images/road_network.png)
+
+This graph represents a road network with 7 cities (A, B, C, and so on) and 11 one-way roads. Each road has a distance (say, in kilometers) and trip time (say, in minutes).
+
+Let's create the graph.
+
+{{< highlight bash >}}
+GRAPH.QUERY g "CREATE (a:City{name:'A'}), (b:City{name:'B'}), (c:City{name:'C'}), (d:City{name:'D'}), (e:City{name:'E'}), (f:City{name:'F'}), (g:City{name:'G'}), (a)-[:Road{time:4, dist:3}]->(b), (a)-[:Road{time:3, dist:8}]->(c), (a)-[:Road{time:4, dist:2}]->(d), (b)-[:Road{time:5, dist:7}]->(e), (b)-[:Road{time:5, dist:5}]->(d), (d)-[:Road{time:4, dist:5}]->(e), (c)-[:Road{time:3, dist:6}]->(f), (d)-[:Road{time:1, dist:4}]->(c), (d)-[:Road{time:2, dist:12}]->(f), (e)-[:Road{time:5, dist:5}]->(g), (f)-[:Road{time:4, dist:2}]->(g)"
+ {{< / highlight >}}
+
+If you're using RedisInsight v2, you can create and visualize the graph by slightly modifying the above query: you'll have to assign aliases to all nodes and relationships, and return them:
+
+{{< highlight bash >}}
+GRAPH.QUERY g "CREATE (a:City{name:'A'}), (b:City{name:'B'}), (c:City{name:'C'}), (d:City{name:'D'}), (e:City{name:'E'}), (f:City{name:'F'}), (g:City{name:'G'}), (a)-[r1:Road{time:4, dist:3}]->(b), (a)-[r2:Road{time:3, dist:8}]->(c), (a)-[r3:Road{time:4, dist:2}]->(d), (b)-[r4:Road{time:5, dist:7}]->(e), (b)-[r5:Road{time:5, dist:5}]->(d), (d)-[r6:Road{time:4, dist:5}]->(e), (c)-[r7:Road{time:3, dist:6}]->(f), (d)-[r8:Road{time:1, dist:4}]->(c), (d)-[r9:Road{time:2, dist:12}]->(f), (e)-[r10:Road{time:5, dist:5}]->(g), (f)-[r11:Road{time:4, dist:2}]->(g) return a,b,c,d,e,f,g,r1,r2,r3,r4,r5,r6,r7,r8,r9,r10,r11"
```

Comment:
1. consider renaming road `i` to its src and dest e.g. `r7` -> `cf`

2. `return a,b,c,d,e,f,g,r1,r2,r3,r4,r5,r6,r7,r8,r9,r10,r11"` -> `RETURN *`

Reply:
Nermina - I updated (on confluence) the query to "RETURN *" (indeed, shorter than the explicit list). Also updated the screenshot.
As for the relationship names - I prefer as it is for clarity.

Reply:
sounds good!